fecgov/openFEC
# Upgrade flyway to v9.19.4

## Summary 

- Resolves #https://github.com/fecgov/openFEC/issues/5465

Upgrade flyway to latest v9.19.4

### Required reviewers

One developer

## Impacted areas of the application

-  Database migrations


## How to test

-  checkout develop
- run `snyk test --file=data/flyway/build.gradle` (snyk flags org.xerial:sqlite-jdbc Arbitrary Code Execution vulnerability)
- checkout feature/5465-upgrade-flyway
- run `brew install flyway` (v9.19.4 will be downloaded)
- run `flyway -v` (verify the downloaded flyway v9.19.4)
- run `dropdb cfdm_test`
- run `createdb cfdm_test`
- run `flyway migrate`
- run `snyk test --file=data/flyway/build.gradle` (snyk no longer flags snyk flags org.xerial:sqlite-jdbc Arbitrary Code Execution vulnerability)
- run `pytest` (all tests PASS)
